specification/glossary.md

```diff
@@ -0,0 +1,50 @@
+# Glossary
+
+Below are a list of some OpenTelemetry specific terms that are used across this
+specification.
+
+## (Telemetry) Library
+
+Denotes the actual *OpenTelemetry* (or API compatible) library in use.  
+See [Library](data-resource-semantic-conventions.md#library)
+
+<a name="instrumented_library"></a>
+
+## Instrumented Library
+
+Denotes the library whose "actions" (e.g. called methods/functions, operations, ...)
+result in OpenTelemetry signals (traces, metrics, logs).  
```

Comment:
This is confusing to me. The only function call that results in a trace for example is `StartSpan` and the library that calls StartSpan is the instrument**ing** library. I see the next sentence clarifies that, but still.

Suggestion: "The library of interest that signals are emitted for (traces, metrics, logs). The library that is monitored."